docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -85,17 +84,17 @@ The `ansible_host` variable reflects the host a task is delegated to.
 Delegating facts
 ----------------
 
-Delegating Ansible tasks is like delegating tasks in the real world - your groceries belong to you, even if someone else delivers them to your home. Similarly, any facts gathered by a delegated task are assigned by default to the `inventory_hostname` (the current host), not to the host which produced the facts (the delegated to host). To assign gathered facts to the delegated host instead of the current host, set `delegate_facts` to `True`::
+Delegating Ansible tasks is like delegating tasks in the real world - your groceries belong to you, even if someone else delivers them to your home. Similarly, any facts gathered by a delegated task are assigned by default to the `inventory_hostname` (the current host), not to the host which produced the facts (the delegated to host). To assign gathered facts to the delegated host instead of the current host, set ``delegate_facts`` to C(true)::
```

Comment:
C() doesn't work in RST files

Reply:
@samccann thanks for pointing this out, filxed